lib/diagnostic.rb

```diff
@@ -20,7 +20,7 @@
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'def star_wars_episode_viii'
```

Comment:
No need for `def` if it's just a variable name. `def` is only used for defining methods in Ruby.